### DISCUSSION TITLE
hubserving方式部署时，出现下面错误，应该怎么处理

### DISCUSSION THREAD

**Discussion by asenasen123** [PaddlePaddle/PaddleOCR#14225] [Q&A]
{
  "msg": "(NotFound) (head_number) is not found in AttributeMap and RuntimeAttributeMap of (matmul) operator.\n  [Hint: Expected iter != op_.RuntimeAttrs().end(), but received iter == op_.RuntimeAttrs().end().] (at ../paddle/fluid/framework/operator.h:463)\n  [operator < matmul > error]",
  "results": "",
  "status": "101"
}

**Comment by GreatV** [comment]
hubserving 已经不再维护，建议不要使用。
